### PR TITLE
Test that canton 3.x builds with bazel hourly

### PR DESCRIPTION
**Pull request by paulbrauner-da**
- creates a canton-3x directory, which will eventually contain the code of the main branch via code drop, but for now only contains the bazel file
- adds an hourly job that checks out our repo, pulls canton main into that directory, runs bazel build canton-daml3/... , and reports errors
- these steps are captured in a shell script run by the hourly job, but that one can also run locally to reproduce the error

### CODE REVIEW

**Comment by paulbrauner-da** [comment]
/azp run

the linux machine got killed :shrug:

**Comment by garyverhaegen-da** [comment]
Bit of a nitpick, but should we call them `canton2` and `canton3` rather than `canton` and `canton3`?

File: canton-3x/shade_rule [hunk 1]
```diff
@@ -0,0 +1 @@
+zap org.apache.pekko.stream.scaladsl.BroadcastHub*
```
Comment (remyhaemmerle-da) [inline canton-3x/shade_rule:1]: If possible I prefer to have final empty line.

**Comment by paulbrauner-da** [comment]
> Bit of a nitpick, but should we call them `canton2` and `canton3` rather than `canton` and `canton3`?

@garyverhaegen-da I agree but I'd like to do this in a separate PR to cut down the noise in this one.

File: canton-3x/shade_rule (hunk 1 above)
Comment (paulbrauner-da) [reply canton-3x/shade_rule]: Done.

**Review by akrmn — APPROVED** [review]
Looks great, thank you @paulbrauner-da!

File: ci/tell-slack-failed.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@ steps:
            -H 'Content-type: application/json' \
            --data "{\"text\":\"$WARNING\n\"}" \
            $(Slack.ci-failures-daml)
+    name: tell_slack_failed
```
Comment (garyverhaegen-da) [inline ci/tell-slack-failed.yml:26]: Thank you!

**Review by garyverhaegen-da — APPROVED** [review]
LGTM.

**Pull request merged**